pages/profile/read/views/profile.jsx

```diff
@@ -75,23 +75,17 @@ class Profile extends React.Component {
 
     const allowedActions = this.props.allowedActions || [];
 
-    const activeProjects = projects.filter(project => {
-      if (project.status !== 'active') {
+    const filterProjects = status => project => {
+      if (project.status !== status) {
         return false;
       }
       const isPrimaryEstablishment = project.establishmentId === establishment.id;
-      const isAdditionalAvailability = project.additionalEstablishments.find(e => e.id === establishment.id);
+      const isAdditionalAvailability = project.additionalEstablishments && project.additionalEstablishments.find(e => e.id === establishment.id);
       return isPrimaryEstablishment || isAdditionalAvailability;
-    });
+    }
 
-    const draftProjects = projects.filter(project => {
-      if (project.status !== 'inactive') {
-        return false;
-      }
-      const isPrimaryEstablishment = project.establishmentId === establishment.id;
-      const isAdditionalAvailability = project.additionalEstablishments.find(e => e.id === establishment.id);
-      return isPrimaryEstablishment || isAdditionalAvailability;
```

Comment:
This logic was repeated twice with only the status changed, so consolidated into the `filterProjects` function above.